# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/quota/test/gtest/QuotaManagerTestHelpers.cpp
# Commit: d586b6b63319
# Full Hash: d586b6b633191a7c6ff251361b0442d98d06c764
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2025-06-15 09:03:40
# Regressor Bug: 1962133
# File Overlap Count: 14
# Description:
#   Bug 1962133 - QM: Add accessed flag to FullOriginMetadata and persist it in the origin metadata file; r=dom-storage-reviewers,hsingh
#   
#   This patch introduces a new mAccessed field in OriginStateMetadata, which is
#   inherited by FullOriginMetadata. One of the reserved 32-bit fields in the
#   metadata header is now used as a bitfield, with one bit allocated to represent
# ==============================================================================

diff -r ca3bdda9fbfd -r d586b6b63319 dom/quota/test/gtest/QuotaManagerTestHelpers.cpp
--- a/dom/quota/test/gtest/QuotaManagerTestHelpers.cpp	Sat Jun 14 02:29:19 2025 +0000
+++ b/dom/quota/test/gtest/QuotaManagerTestHelpers.cpp	Sat Jun 14 02:46:33 2025 +0000
@@ -38,7 +38,8 @@
                                          const nsCString& aGroupNoSuffix,
                                          const nsCString& aOriginNoSuffix) {
   return {GetOriginMetadata(aOriginSuffix, aGroupNoSuffix, aOriginNoSuffix),
-          OriginStateMetadata{/* aLastAccessTime */ 0, /* aPersisted */ false}};
+          OriginStateMetadata{/* aLastAccessTime */ 0, /* aAccessed */ false,
+                              /* aPersisted */ false}};
 }
 
 }  // namespace mozilla::dom::quota::test